doc/classes/RichTextLabel.xml

```diff
@@ -520,6 +520,7 @@
 		</member>
 		<member name="bbcode_enabled" type="bool" setter="set_use_bbcode" getter="is_using_bbcode" default="false">
 			If [code]true[/code], the label uses BBCode formatting.
+			[b]Warning:[/b] Be careful about user input when [member bbcode_enabled] is [code]true[/code], as users may be able to inject arbitrary BBCode tags. This can be a security concern, since users could break formatting or create clickable links to malicious websites. Use [method String.bbcode_escape] when adding text to prevent users from injecting arbitrary BBCode tags.
```

Comment:
```suggestion
			[b]Warning:[/b] Be careful about user input when [member bbcode_enabled] is [code]true[/code], as users may be able to inject arbitrary BBCode tags. This can be a security concern, since users could break formatting or create clickable links to malicious websites. Use [method String.bbcode_escape] or [method String.strip_bbcode] when adding text to prevent users from injecting arbitrary BBCode tags.
```